### DISCUSSION TITLE
Why can't I enter when executing?

### DISCUSSION THREAD

**Discussion by xskaiYT** [mitmproxy/mitmproxy#6331] [Q&A]
![image](https://github.com/mitmproxy/mitmproxy/assets/138776702/e4596798-16d5-4642-b713-7178a8e3acf8)

I want to type in it, but there is no way

help

**Comment by ThomasBucaioni** [comment]
Start browsing on the Internet, and when you're done press `Ctrl-c`. Maybe you'd like to interact with it? Try to run `mitmproxy` instead of `mitmdump`
